### ISSUE TITLE
Expose arcan

### ISSUE THREAD

**Issue by fricklerhandwerk** [ngi-nix/ngipkgs#454]
* [arcan](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/ar/arcan/package.nix)

Specifically A12 protocol libraries and arcan-net server/client binary, which is [part of NlNet project](https://nlnet.nl/project/Arcan-A12/). The package builds the whole arcan project though.

_Originally posted by @cipharius in https://github.com/ngi-nix/ngipkgs/issues/76#issuecomment-2152268983_

**Comment by OluchiTheAnalyst** [comment]
Hi @eljamm is this still Open?

**Comment by eljamm** [comment]
Yes, but if you want to pick this up, it would be better if you [triage the project](https://github.com/ngi-nix/ngipkgs/issues/721) first. Should I assign you for the triage?

**Comment by OluchiTheAnalyst** [comment]
Yes, Sure💯

**Comment by OluchiTheAnalyst** [comment]
@eljamm Please could you point me to any documentation or resources that could guide me through the process of exposing arcan?
Thank you for your support!

**Comment by eljamm** [comment]
The process of exposing a project is similar to [migrating](https://github.com/ngi-nix/ngipkgs/issues/593) an existing one. The only difference is that we're adding the project metadata for the first time. To do this:

- Copy the template from `templates/project` to `projects/<PROJECT_NAME>`
- Using the [triaged data](https://github.com/ngi-nix/ngipkgs/issues/721), fill in some basic information about the project (programs, services, example, tests, links, ...)

The information you add doesn't need to be exhaustive, we just want to have an entry which we can build on.

**Comment by OluchiTheAnalyst** [comment]
Hi @eljamm I just created a PR [Here](https://github.com/ngi-nix/ngipkgs/pull/745).
Please could you have a look?

**Comment by eljamm** [comment]
Done in https://github.com/ngi-nix/ngipkgs/pull/745
